indexing-solr/src/test/java/info/rmapproject/indexing/solr/TestResourceManager.java

```diff
@@ -14,6 +14,7 @@
 import java.util.Set;
 
 import static info.rmapproject.indexing.IndexUtils.iae;
+import info.rmapproject.indexing.TestUtils;
```

Comment:
This import is unused

Reply:
I believe it is used... perhaps when TestUtils was in the same package it didn't have to be imported? Not sure, but there are references to it.